workspace-optimizer.Dockerfile

```diff
@@ -10,8 +13,8 @@ RUN python3 --version
 
 # Install Rust nightly
 # Choose version from: https://rust-lang.github.io/rustup-components-history/x86_64-unknown-linux-gnu.html
-RUN rustup toolchain install nightly-2020-10-14 --allow-downgrade --profile minimal --target wasm32-unknown-unknown
-RUN rustup default nightly-2020-10-14
+#RUN rustup toolchain install nightly-2020-10-14 --allow-downgrade --profile minimal --target wasm32-unknown-unknown
+#RUN rustup default nightly-2020-10-14
```

Comment:
`workspace-optimizer` requires nightly (unfortunalety) in order to use `--out-dir`. Without that we need to do a horrible search in the the target dir for compilation results.

Reply:
OK. Will update it to latest nigthly then.